app/views/posts/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1><%= @post.title %></h1>
+<div><em>
+  by
+  <%= link_to user_path(@post.user) do %>
+    <%= @post.user.name-%>
+  <% end %>
+  |
+  <%= render partial: "likes/like_button", locals: { like: Like.new, user: @post.user, post: @post } %>
+  |
+  <span><%= @post.comments_counter || 0 %> Comments</span>
+</em></div>
+<p><%= @post.text %></p>
+
+<hr />
+
+<div class="comments-wrapper">
+  <h3>Comments:</h3>
+  <% if @post.comments.empty? %>
+    <p>There no comment so far!</p>
+  <% end %>
+  <% @post.comments.each do |comment| %>
+    <div class="comment-card">
+      <strong>
+        <%= link_to user_path(comment.user) do %>
+          <%= comment.user.name %>
+        <% end %>
+        :
+      </strong>
+      <%= comment.text %>
+    </div>
+  <% end %>
+
```

Comment:
- [ ] KIndly fix the error on this page

![image](https://user-images.githubusercontent.com/65068771/179355091-f1d8ed9f-2c95-4c05-83bf-4c92a5bd3b92.png)